adoorback/account/models.py

```diff
@@ -337,6 +338,33 @@ def can_access_note_set(self, user):
         return note_queryset
 
 
+#added 11/18/24:
+    def get_subscribed_content(self, content_type=None):
+        from category.models import Subscription
+        subscriptions = Subscription.objects.filter(user=self)
+        scopes = [sub.sharing_scope for sub in subscriptions]
+        
+        if content_type == 'response':
+            from qna.models import Response
+            return Response.objects.filter(sharing_scope__in=scopes)
+        elif content_type == 'note':
+            from note.models import Note
+            return Note.objects.filter(sharing_scope__in=scopes)
+        else:
+            from qna.models import Response
+            from note.models import Note
+            responses = Response.objects.filter(sharing_scope__in=scopes)
+            notes = Note.objects.filter(sharing_scope__in=scopes)
+            return responses, notes
+
+    def get_unread_subscribed_content(self):
```

Comment:
Ah I thought it might be useful for seeing if a user had read subscribed content or not, but I think it may not be necessary now -- I might have been overthinking. I can remove it!

Reply:
Oh, I just wanted to clarify this to check if it works as intended. :)
Alright, you can rebuild it when you need it again later!